grid-proxy/internal/explorer/db/postgres.go

```diff
@@ -98,19 +98,34 @@ const (
 		END;
 	RETURN v_dec_value;
 	END;
-	$$ LANGUAGE plpgsql;`
+	$$ LANGUAGE plpgsql;
+	
+	CREATE OR REPLACE FUNCTION notify_node_change() RETURNS trigger AS $notify_node_change$
+	BEGIN
+		PERFORM pg_notify('node_added', NEW.twin_id);
+		RETURN NULL;
+	END;
+	$notify_node_change$ LANGUAGE plpgsql;
+	
+	CREATE OR REPLACE TRIGGER node_added
+	AFTER INSERT ON node
+	FOR EACH ROW EXECUTE PROCEDURE notify_node_change();
+	`
 )
 
 // PostgresDatabase postgres db client
 type PostgresDatabase struct {
 	gormDB *gorm.DB
 }
 
-// NewPostgresDatabase returns a new postgres db client
-func NewPostgresDatabase(host string, port int, user, password, dbname string) (Database, error) {
-	psqlInfo := fmt.Sprintf("host=%s port=%d user=%s "+
+func GetConnectionString(host string, port int, user, password, dbname string) string {
+	return fmt.Sprintf("host=%s port=%d user=%s "+
```

Comment:
why sslmode is disabled?

Reply:
it was always like this, should we enable it?